server/src/main/java/io/seata/server/storage/file/lock/FileLocker.java

```diff
@@ -75,16 +75,16 @@ public boolean acquireLock(List<RowLock> rowLocks, boolean autoCommit, boolean s
         String resourceId = branchSession.getResourceId();
         long transactionId = branchSession.getTransactionId();
 
-        ConcurrentMap<BucketLockMap, Set<String>> bucketHolder = branchSession.getLockHolder();
-        ConcurrentMap<String, ConcurrentMap<Integer, BucketLockMap>> dbLockMap = CollectionUtils.computeIfAbsent(
-            LOCK_MAP, resourceId, key -> new ConcurrentHashMap<>());
+        Map<BucketLockMap, Set<String>> bucketHolder = new ConcurrentHashMap<>(8);
```

Comment:
Why assemble logic outside instead of cohesion logic in `BranchSession`? I think the `lockHolder` should be final.

---

为什么在外面组装逻辑而不是在BranchSession内聚逻辑。
我认为lockHolder应该是final的。